filterprotobuffer_test.go

```diff
@@ -0,0 +1,97 @@
+package dapper_test
+
+import (
+	"github.com/dogmatiq/dapper"
+	"github.com/dogmatiq/dapper/internal/fixtures"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+)
+
+// matchProtoFormat works around non-deterministic behaviour introduced by the protobuf package.
+// They want to enforce nobody relies on the output being stable. It's not totally unreasonable but this library
+// is all about the output, so we really want to know when it changes.
+// See https://github.com/golang/protobuf/issues/1121
```

Comment:
```suggestion
// matchProtoFormat works around non-deterministic behaviour introduced by the protobuf package.
//
// The maintains of the Go protocol buffers library want to emsure nobody relies on the output being stable. It's not totally unreasonable but Dapper
// is all about the output, so we really want to know when it changes.
//
// See https://github.com/golang/protobuf/issues/1121
```

Edited for formatting and a bit of clarity about the parties involved.